test/index.test.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const Bookshelf = require('bookshelf')(require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  }
+}));
```

Comment:
I think I agree with @robinjoseph08 on this. It might get annoying if the file changes every time a different dev works on this project (due to any data/schema changes, or even just different versions of sqlite writing differently).

It's probably okay to include it for now, but I wouldn't be surprised if we end up `gitignore`'ing it in the near future.